lib/srv/usermgmt.go

```diff
@@ -303,6 +312,7 @@ func (u *HostUserManagement) createGroupIfNotExist(group string) error {
 // See github issue - https://github.com/golang/go/issues/40334
 func isUnknownGroupError(err error, groupName string) bool {
 	return errors.Is(err, user.UnknownGroupError(groupName)) ||
+		errors.Is(err, user.UnknownGroupIdError(groupName)) ||
```

Comment:
```suggestion
		errors.Is(err, user.UnknownGroupIDError(groupName)) ||
```

Reply:
Thats from the os/user package i cant change it